versions.tf

```diff
@@ -3,12 +3,12 @@
 # ----------------------------------------------------------------------------------------------------------------------
 
 terraform {
-  required_version = "~> 1.0"
+  required_version = ">= 0.14.7, < 2.0"
 
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "~> 4.0"
+      version = ">= 3.50, < 5.0"
```

Comment:
please keep the minimum version as previously released. We do not add support for previous major versions anymore.